src/components/BlockchainList/TwitterLink.tsx

```diff
@@ -5,33 +5,33 @@ import Delimiter from 'components/Delimiter'
 import postIdsStatuses from 'stores/PostIdsStatuses'
 
 interface TwitterLinkProps {
-  id: number
+  blockchainId: number
 }
 
-function TwitterLinkSuspended({ id }: TwitterLinkProps) {
-  const { currentStatuses } = useSnapshot(postIdsStatuses)
-  const statusId = currentStatuses[id]?.statusId
+function TwitterLinkSuspended({ blockchainId }: TwitterLinkProps) {
+  const currentStatuses = useSnapshot(postIdsStatuses.currentStatuses)
```

Comment:
Why did you change it from `const { currentStatuses } = useSnapshot(postIdsStatuses)`?